site/components/Navigation/index.tsx

```diff
@@ -58,47 +64,345 @@ export const Navigation: FC<Props> = ({
         />
         <NavItemDesktop
-          url="/blog"
           name={t({ message: "Resources", id: "shared.resources" })}
           active={activePage === "Resources"}
+          subMenu={[
+            <NavItemDesktop
+              name={t({
+                message: "HOW TO BUY KLIMA",
+                id: "shared.how_to_buy",
+              })}
+              key="how to buy"
+              href="/buy"
+            />,
+            <NavItemDesktop
+              name={t({
+                message: "CARBON DASHBOARDS",
+                id: "shared.carbon_dashboards",
+              })}
+              key="carbon dashboards"
+              href={createLinkWithLocaleQuery(urls.carbonDashboard, locale)}
+            />,
+            <NavItemDesktop
+              name={t({
+                message: "DISCLAIMER",
+                id: "shared.disclaimer",
+              })}
+              key="disclaimer"
+              href="/disclaimer"
+            />,
+            <NavItemDesktop
+              name={t({
+                message: "DOCS",
+                id: "shared.docs",
+              })}
+              key="docs"
+              href={createLinkWithLocaleQuery(urls.officialDocs, locale)}
+            />,
+          ]}
         />
       </HeaderDesktop>
 
       <HeaderMobile
-        buttons={[
-          <ChangeLanguageButton key="ChangeLanguageButton" />,
-          ...(showThemeToggle ? [<ThemeToggle key="ThemeToggle" />] : []),
-        ]}
+        buttons={[<ChangeLanguageButton key="ChangeLanguageButton" />]}
       >
-        <NavItemMobile
-          url={createLinkWithLocaleQuery(urls.offset, locale)}
-          name={t({ message: "Offset", id: "shared.offset" })}
-        />
-        <NavItemMobile
-          url="/infinity"
-          name={t({ message: "Infinity", id: "shared.infinity" })}
-        />
-        <NavItemMobile
-          url={urls.loveletter}
-          name={t({ message: "Love Letters", id: "shared.loveletters" })}
-        />
-        <NavItemMobile
-          url="/blog"
-          name={t({ message: "Resources", id: "shared.resources" })}
-        />
+        <div className={styles.navMain_MobileItemsWrapper}>
+          <div className="links">
+            <NavItemMobile
+              name={t({ message: "About", id: "shared.about" })}
+              toggledNavItemID={toggledNavItemID}
+              setToggledNavItemID={setToggledNavItemID}
+              id="About"
+              subMenu={[
+                <NavItemMobile
+                  name={t({
+                    message: "BLOG",
+                    id: "shared.blog",
+                  })}
+                  key="blog"
+                  href="/blog"
+                />,
+                <NavItemMobile
+                  name={t({
+                    message: "COMMUNITY",
+                    id: "shared.community",
+                  })}
+                  key="community"
+                  href="/community"
```

Comment:
Now you can pass the check for any active page on SITE to each submenu item too (which links to a SITE page).
And add styles accordingly.

```TS
<NavItemMobile
     url="/community"
    active={activePage === "community"}
     .....
```

Reply:
added to all site nav items